### DISCUSSION TITLE
Selektion erweitern/einschränken diverses

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#158] [General]
Wenn man nach einer Suche/Abfrage die Selektion erweitert/einschränkt
 
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/91e5b7dd-46aa-4620-a2df-7fc2e69d85fe)

und danach das Ergebnis entfernt
 
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/dbe8ac32-a7c8-40c0-b7aa-1ea95d8d287c)

ist auf einmal das Standartwerkzeug "pan"
 
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/d8537161-564c-471c-b949-bc78be2e5546)

Und es besteht keine Möglichkeit auf "Abfrage von sichtbaren Objekten".
Erst mit Aufrufen und Schließen eines Tools erscheint wieder die Zeigefingerhand mit dem blauen Kreis und weißem i.
 
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/c44f7f33-d5b3-4432-9655-9a3f48e7cfc2)

Und draufgekommen bin ich, weil ich da mit den Toleranzen für Polygone, Linien und Punkten ausprobieren wollte.
 
![image](https://github.com/gis-eni/webgis-forum/assets/1842500/af8563cf-8501-4978-9e16-10832f510eaf)

Funktioniert gut bei der Abfrage … sobald man aber "Selektion erweitern/einschränken" ausführt, wird die 0-Toleranz für Polygone ignoriert.
lg
marko

**Comment by gis-eni** [comment]
Beide "Bugs" konnte ich nachvollziehen und sind mit dem nächsten Build behoben. Mit ist dann noch aufgefallen, dass es eine Tippfehler gibt. Es müsste heißen ``tolerance-for-polygon-layers`` (nicht ``polygone``).
Auch diesen Fehler hab ich ausgebessert, darum must du das beim nächsten Build auch in deiner ``api.config`` ändern.
